pre_processing/rttov_driver.F90

```diff
@@ -929,7 +932,7 @@ subroutine rttov_driver(coef_path, emiss_path, granule, preproc_dims, &
          call rttov_deallocate_emis_atlas(emis_atlas)
          call rttov_alloc_traj(stat, 1, nchan, opts, nlevels, coefs, DEALLOC, &
               traj)
-         call rttov_alloc_transmission(stat, transmission, nlevels, nevals, &
+         call rttov_alloc_transmission(stat, transmission, nlevels, nchan, &
```

Comment:
This also needs to be done in `rttov_driver_gfs.F90`, which Simon added for reasons I have never bothered to question despite it repeating a fuckton of code.

Reply:
As with `rttov_driver.F90`, `nevals` is no more.

There doesn't seem to be much difference between `rttov_driver.F90` and `rttov_driver_gfs.F90` - perhaps a conversation about whether it is needed could be had....